<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSelectionDialog</class>
 <widget class="QDialog" name="ColorSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose colors...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>The square detector will look for squares of the following colors:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="colorList"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>New color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="colorBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addBtn">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeSelectedBtn">
         <property name="text">
          <string>Remove selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeAllBtn">
         <property name="text">
          <string>Remove all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetBtn">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveBtn">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
